study.md

```diff
@@ -49,15 +49,16 @@ should contain two strings, "drinking beer" and "conspiracy theories".  The keys
 should be symbols.
 
 ```ruby
-#your answer here
+dale = Hash.new
+dale["middle name"=> "Alvin", "hobbies"=>["drinking beer","conspiracy theories" ]]
```

Comment:
This will result in a syntax error because it will interpret `dale` as an array and `"middle name"=> "Alvin", "hobbies"=>["drinking beer","conspiracy theories" ]` as an array index.